web_src/js/features/comp/EasyMDE.js

```diff
@@ -74,10 +74,10 @@ export async function createCommentEasyMDE(textarea, easyMDEOptions = {}) {
   const inputField = easyMDE.codemirror.getInputField();
   inputField.classList.add('js-quick-submit');
   easyMDE.codemirror.setOption('extraKeys', {
-    Enter: () => {
+    Enter: (cm) => {
       const tributeContainer = document.querySelector('.tribute-container');
       if (!tributeContainer || tributeContainer.style.display === 'none') {
-        return window.CodeMirror.Pass;
+        cm.execCommand('newlineAndIndent');
```

Comment:
Why not `cm.Pass`?

Reply:
`CodeMirror.Pass` exists, while `cm = CodeMirror(...); cm.Pass` doesn't exist.

I have read through the EasyMDE and CodeMirror code, it's impossible to access `CodeMirror.Xxx` in Gitea now.

Reply:
Fine I guess if the end result is the same.

Reply:
It is the same. CodeMirror document said: the default command for `Enter` is `newlineAndIndent`.

https://codemirror.net/doc/manual.html

<img width="658" alt="image" src="https://user-images.githubusercontent.com/2114189/156881864-ee70f6cd-93a2-45d5-88d3-18b886ec5cbf.png">